pre_commit/commands/autoupdate.py

```diff
@@ -14,6 +14,7 @@
 from pre_commit.clientlib import is_local_repo
 from pre_commit.clientlib import is_meta_repo
 from pre_commit.clientlib import load_config
+from pre_commit.clientlib import InvalidManifestError
```

Comment:
`pre-commit install` and then `pre-commit run --all-files` ;) (it'll fix the import ordering issue for you automatically, and enforce it on commits after that!)